index.html

```diff
@@ -438,6 +454,7 @@ <h2>
         />
```

Comment:
- [ ] Please delete this closing tag and the anchor that was part of the commented code at the top.